config/field_test.yml

```diff
@@ -28,10 +28,38 @@
 ################################################################################
 experiments:
   # NOTE: Our feed strategy testing experiment must begin with "feed_strategy"
+  feed_strategy_starting_20230424:
+    # NOTE: Required as we want only want to consider for conversion events that
+    #       occurred on or after the given start_date.
+    started_at: 2023-04-24
+    variants:
+      - 20230110-variant-c
+      - 20230424-variant-a
+      - 20230424-variant-b
+      - 20230424-variant-c
+    weights:
+      - 55
+      - 15
+      - 15
+      - 15
+    goals:
```

Comment:
Removed some unnecessary goals here to focus on fewer that are more likely to be relevant. Still lots of variations in the goals.